README.md

```diff
@@ -83,7 +88,18 @@ This removes all docker images, networks, and local volumes.
       * the **CHPL querier**, which requests health IT product information from CHPL and adds these to the database
       * the **NPPES endpoint populator**, which adds endpoint data from the monthly NPPES export to the database. 
       * the **NPPES org populator**, which adds provider data from the monthly NPPES export to the database. 
-        * this is item will take an hour to load if you use the full npidata_pfile
+
+    If you wish to run Lantern with the production resources, you must run the following command to query NPPES for their endpoint and npi data files and automatically populate the database with this information, as the files are too large to be persisted in our list of resources, as well as populate the database using the data found in `lantern-back-end/resources/prod_resources`. 
```

Comment:
Technically this isn't true, if you run `make run` and then run `make populatedb_prod`, then it will populate the database with the data from dev_resources because of how the docker-compose files work. I think it's okay to work like this, but we should probably make a note here and below when we add `make populatedb` and `make populatedb_prod` to the Make Command section that it is dependent on what version of the environment you are running.